src/map.cpp

```diff
@@ -6753,7 +6753,7 @@ void map::rotten_item_spawn( const item &item, const tripoint &pnt )
     if( mgroup == "GROUP_NULL" ) {
         return;
     }
-    const int chance = ( comest->rot_spawn_chance * get_option<int>( "CARRION_SPAWNRATE" ) ) / 100;
+    const int chance = ( int )( comest->rot_spawn_chance * get_option< float >( "CARRION_SPAWNRATE" ) );
```

Comment:
```suggestion
    const int chance = static_cast<int>( comest->rot_spawn_chance * get_option< float >( "CARRION_SPAWNRATE" ) );
```

Reply:
Are you sure it should be written as <int> and not < int >. I don't like it myself either, but that is what my astyle extension says it should be like. Perhaps it is acting wrong though.

Reply:
That is not an astyle issue - you need to use modern cast, instead of C-style casts (e.g. `static_cast<T>( ... )` instead of `( T ) ...` )

Reply:
That is definitely true. What I was referring to is your other change in that you want to remove the spaces surround the float and by that turn `get_option< float >` into `get_option<float>`.

Reply:
`astyle` allows spaces or not inside template angle brackets.  Generally our code doesn't have spaces, although there are exceptions.

Reply:
Ah, sure. I'll add it without spaces then.

Reply:
Now this line is too long.

Reply:
@AMurkin Thanks, that explains it. It should be solved with the latest commits.